packages/react-magma-dom/src/components/NativeSelect/NativeSelect.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Card } from '../Card';
+import { CardBody } from '../Card/CardBody';
+import { magma } from '../../theme/magma';
+import { NativeSelect, NativeSelectProps } from '.';
+import { Story, Meta } from '@storybook/react/types-6-0';
+
+const Template: Story<NativeSelectProps> = args => (
+  <NativeSelect {...args}>
+    <option>Red</option>
+    <option>Green</option>
+    <option>Blue</option>
+  </NativeSelect>
+);
+
+export default {
+  component: NativeSelect,
+  title: 'NativeSelect',
+  argTypes: {
+    isInverse: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+} as Meta;
+
+export const Default = Template.bind({});
+Default.args = {
+  isInverse: false,
+  labelText: 'Select',
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  ...Default.args,
+  disabled: true,
+};
+
+export const HasError = Template.bind({});
+HasError.args = {
+  ...Default.args,
+  hasError: true,
```

Comment:
User shouldn't need to specify `hasError` if they are also providing an errorMessage.